### PR TITLE
the integration test make target was wrong

### PR DESCRIPTION
**Pull request by AymericPlanche**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Review by mr-prud — APPROVED** [review]
GG! Good analyze for a clean fix! 👏🏿

**Review by greg-akeneo — APPROVED** [review]

File: make-file/test.mk [hunk 1]
```diff
@@ -68,8 +68,8 @@ acceptance-front:
 integration-front:
 	$(YARN_RUN) integration
 
-.PHONY: integration-back
-integration-back: var/tests/phpunit connectivity-connection-integration-back communication-channel-integration-back
+.PHONY: pim-integration-back
```
Comment (jmleroux) [inline make-file/test.mk:71]: We should be careful before renaming targets and check if they are not used elsewhere (Onbarder)

@damien-carcel ?

**Review by jmleroux — COMMENTED** [review]
I thought i already fixed it here: https://github.com/akeneo/pim-enterprise-dev/commit/21a1a54c69eb8b42cd04259dd68be464b9130f2c

:thinking:

File: make-file/test.mk (hunk 1 above)
Comment (damien-carcel) [reply make-file/test.mk:71]: Indeed, this will break our bundle CI.

Comment (damien-carcel) [reply make-file/test.mk:71]: By the way, I don't understand what the issue is with the original make command name…

Comment (AymericPlanche) [reply make-file/test.mk:71]: @damien-carcel: about the naming "IDK :)" 
@damien-carcel did you see the analyse on slack to understand the why ?

Comment (damien-carcel) [reply make-file/test.mk:71]: Ah, OK, no, I didn't see it :slightly_smiling_face:

Comment (AymericPlanche) [reply make-file/test.mk:71]: seen with damien. I'll merge and backport in 5.0

**Pull request merged**